Kitodo/src/test/java/org/kitodo/test/utils/TestConstants.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.test.utils;
+
+/**
+ * This class is used to group constants used in various tests in one place from which they can be retrieved.
+ */
+public class TestConstants {
```

Comment:
Maybe define this class as `public final class TestConstants`  like in https://www.baeldung.com/java-constants-good-practices described? There are some more notices to constant classes.